tests/test_plots.py

```diff
@@ -49,3 +49,15 @@ def test_two_models_5_tries(self):
 
     with plt.ion():
       plot_benchmarks(misclass_rates, 'k', benchmark_range=[10, 25, 50, 100])
+
+  def test_modes(self):
+    misclass_rates = {
+      'Supervised Marker Map': np.array([[0.18801997, 0.23460899]]),
+      'Mixed Marker Map': np.array([[0.19966722, 0.26622296]])
+    }
+
+    with plt.ion():
```

Comment:
What does plt.ion do here? Is this `with` so that the plots are updated as these, possibly long running, benchmarks complete?